client/webserver/locales/de-de.go

```diff
@@ -244,4 +245,20 @@ var DeDE = map[string]string{
 	"export_wallet_msg":           "Nachfolgend findest du die erforderlichen Seeds um dein Wallet in einigen der beliebten externen Wallets wiederherzustellen. Führe KEINE Transaktionen mit deinen externen Wallet durch während du aktive Trades auf dem DEX laufen hast.",
 	"clipboard_warning":           "Das Kopieren/Einfügen eines Wallet-Seeds stellt ein potenzielles Sicherheitsrisiko dar. Dies geschieht auf eigene Gefahr.",
 	"fiat_exchange_rate_sources":  "Quellen für den Fiat-Wechselkurs",
+	"wallet_wait_synced":          "Das Wallet wird nach der Synchronisierung erstellt",
```

Comment:
@karamble Sorry, I forgot one entirely.

https://github.com/decred/dcrdex/blob/b46222130c4956cfd1f2da1e07301cfcbd64671f/client/webserver/locales/en-us.go#L248

`"Synchronizing"`

Reply:
@karamble Would you provide this one quick?  Thanks.

Reply:
I'm guessing just "synchronisieren" but I really don't know if that's appropriate in this context.

Reply:
correct